native/packages/react-native-bpk-component-button-link/src/styles.js

```diff
@@ -22,28 +22,27 @@ import { StyleSheet } from 'react-native';
 import {
   colorBlue500,
   colorGray300,
-  borderRadiusSm,
   spacingSm,
-  spacingMd,
+  spacingXl,
 } from 'bpk-tokens/tokens/base.react.native';
 
 const styles = StyleSheet.create({
-  container: {
```

Comment:
This was only ever used on iOS. Moreover it added a border radius to something that doesn't have a visible box therefore it is redundant.